### PR TITLE
Vi fix date display in cprs

### PR DESCRIPTION
**Pull request by shabiel**
Includes Vitals Clients source code commit.

### CODE REVIEW

File: Packages/General Medical Record - Vitals/Vitals Clients/Orpheus-4.0.8/source/hpp/Win32/Debug/O32IGrid.hpp [hunk 1]
```diff
@@ -0,0 +1,753 @@
+﻿// CodeGear C++Builder
```
Comment (ChristopherEdwards) [inline Packages/General Medical Record - Vitals/Vitals Clients/Orpheus-4.0.8/source/hpp/Win32/Debug/O32IGrid.hpp]: These hpp files seem to all be auto generated (based on comments within the file), do we need to track them?
Also, the header contains all rights reserved and copyrighted by Embarcadero, I don't think that gives us a right to redistribute, or compatible with the Apache 2.0 license OSEHRA uses.

File: Packages/General Medical Record - Vitals/Vitals Clients/Orpheus-4.0.8/source/dunit/readme.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+This is the DUNIT test project for Orpheus.
```
Comment (ChristopherEdwards) [inline Packages/General Medical Record - Vitals/Vitals Clients/Orpheus-4.0.8/source/dunit/readme.txt]: Side comment: Wow we got tests! Is OSEHRA planning on hooking these up to CDash in the future? (I think OSEHRA has other DUnit tests that were hooked up at one point.

File: Packages/General Medical Record - Vitals/Vitals Clients/Orpheus-4.0.8/packages/Delphi/OrpheusDD.dpk [hunk 3]
```diff
@@ -0,0 +1,102 @@
+{* ***** BEGIN LICENSE BLOCK *****                                            *}
```
Comment (ChristopherEdwards) [inline Packages/General Medical Record - Vitals/Vitals Clients/Orpheus-4.0.8/packages/Delphi/OrpheusDD.dpk]: We should have a change to the NOTICE file indicating certain parts are licensed under MPL.

File: Packages/General Medical Record - Vitals/Vitals Clients/Orpheus-4.0.8/packages/Delphi/OrpheusDBDR.dpk [hunk 4]
```diff
@@ -0,0 +1,86 @@
+{* ***** BEGIN LICENSE BLOCK *****                                            *}
```
Comment (ChristopherEdwards) [inline Packages/General Medical Record - Vitals/Vitals Clients/Orpheus-4.0.8/packages/Delphi/OrpheusDBDR.dpk]: This appears to be where the original source is located: https://sourceforge.net/projects/tporpheus/

**Comment by ChristopherEdwards** [comment]
I think i see visual source safe files (vss) that are part of the diff that we probably don't need.

**Comment by shabiel** [comment]
Chirstopher, I will try to take care of these issues. I will need to
delete most of these files and see whether things still compile. Yes,
I forgot about the NOTICE file.

File: Packages/General Medical Record - Vitals/Vitals Clients/Orpheus-4.0.8/source/dunit/readme.txt (hunk 2 above)
Comment (ChristopherEdwards) [reply Packages/General Medical Record - Vitals/Vitals Clients/Orpheus-4.0.8/source/dunit/readme.txt]: ignore this comment as this is a 3rd party module and doesn't appear to be vitals related.

**Comment by shabiel** [comment]
Hey Chris. I removed the external modules as they were not necessary
for the Vitals DLL to compile. They may be needed for the
client/manager application; but I will cross that bridge if I ever get
to it.

I also reviewed all the files for licensing issues; and all the .pas
files in the repo look like they are unlicensed (public domain).

--Sam

**Pull request merged**